client/modules/IDE/components/Editor/index.jsx

```diff
@@ -146,168 +336,135 @@ class Editor extends React.Component {
       }
     });
 
-    this.hinter = new Fuse(hinter.p5Hinter, {
-      threshold: 0.05,
-      keys: ['text']
-    });
-
-    delete this._cm.options.lint.options.errors;
+    const cm = cmInstance.current;
+    emmet(CodeMirror);
 
     const replaceCommand =
       metaKey === 'Ctrl' ? `${metaKey}-H` : `${metaKey}-Option-F`;
-    this._cm.setOption('extraKeys', {
-      Tab: (cm) => {
-        if (!cm.execCommand('emmetExpandAbbreviation')) return;
-        // might need to specify and indent more?
-        const selection = cm.doc.getSelection();
+    cm.setOption('extraKeys', {
+      Tab: (_cmInstance) => {
+        if (!_cmInstance.execCommand('emmetExpandAbbreviation')) return;
+        const selection = _cmInstance.doc.getSelection();
         if (selection.length > 0) {
-          cm.execCommand('indentMore');
+          _cmInstance.execCommand('indentMore');
         } else {
-          cm.replaceSelection(' '.repeat(INDENTATION_AMOUNT));
+          _cmInstance.replaceSelection(' '.repeat(INDENTATION_AMOUNT));
```

Comment:
not sure why eslint didn't catch this but i don't think this casebase uses underscores! looks like they are following the airbnb style guide: https://github.com/airbnb/javascript?tab=readme-ov-file#naming--leading-underscore

Reply:
I think [this rule is turned off in our eslint configuration](https://github.com/processing/p5.js-web-editor/blob/87b41992ccd39b0ee79b01dc886c5158ce7fdfd4/.eslintrc#L23) 🤔 but it makes sense to me to aim to align with the style guide as much as possible!